.github/workflows/reviews.yml

```diff
@@ -0,0 +1,12 @@
+name: Dismiss old reviews when re-requested
+
+on:
+  pull_request:
+    types: [review_requested]
+  workflow_dispatch:
+
+jobs:
+  review_requested:
+    permissions:
+      pull-requests: write
+    uses: cryspen/actions/.github/workflows/dismiss-old-reviews.yml@main
```

Comment:
Just to verify my understanding is correct, this action dismisses old reviews of exactly those reviewers from which a review has been re-requested. If there are other reviewers from which a review has not been re-requested, their reviews will not be dismissed, right?

Reply:
Yes, that's the idea. We'll need to see if it does the right thing 😄 .